src/GameBoard.hs

```diff
@@ -1,98 +1,93 @@
-  ( winHeight, winWidth, offset
-  , gameWidth, gameHeight
-  , wallWidth
-  , brickWidth, brickHeight
-  , ballRadius
-  , wallUpPos, wallDownPos, wallLeftPos, wallRightPos
-  , Position
-  , Radius
-  , Width
-  , Height
-  , Brick(..)
-  , Game (..)
-  , initialState
-  ) where
+    ( winHeight, winWidth, offset
+    , gameWidth, gameHeight
+    , wallWidth
+    , brickWidth, brickHeight
+    , ballRadius, speedRatio
+    , wallUpPos, wallDownPos, wallLeftPos, wallRightPos
+    , paddleWidth, paddleHeight, paddleStep
+    , Position
+    , Radius
+    , Width
+    , Height
+    , Brick(..)
+    , Game (..)
+    , Paddle (..)
+    , initialState
+    ) where
 
-  import Graphics.Gloss
+import Graphics.Gloss
 
-  -- | position of the window on the screen
-  offset:: Int
-  offset = 100
+-- | position of the window on the screen
+offset:: Int
+offset = 100
 
-  -- | Window width and height
-  winHeight, winWidth :: Int
-  winWidth = floor (gameWidth + wallWidth * 2)
-  winHeight = floor (gameHeight + wallWidth * 2)
+-- | Window width and height
+winHeight, winWidth :: Int
+winWidth = floor (gameWidth + wallWidth * 2)
+winHeight = floor (gameHeight + wallWidth * 2)
 
-  -- | all the widths : brick, game
-  -- wallWidth = thickness
-  gameWidth, wallWidth, brickWidth, paddleWidth :: Width
-  gameWidth = 800
-  wallWidth = 10
-  brickWidth = 100
-  paddleWidth = 100
+-- | all the widths : brick, game, paddle
+-- wallWidth = thickness
+gameWidth, wallWidth, brickWidth, paddleWidth :: Width
+gameWidth = 800
+wallWidth = 10
+brickWidth = 100
+paddleWidth = 100
 
-  -- | speed ration (when the ball hits a brick)
-  speedRatio :: Float
-  speedRatio = 1.5
+-- | speed ration (when the ball hits a brick)
+speedRatio, paddleStep :: Float
+speedRatio = 1.5
+paddleStep = 5
 
-<<<<<<< ef207458608785b7998e8f3d8c39008da8a43327
-  -- | all the widths : brick, wall, game
-  gameHeight, brickHeight, paddleHeight :: Height
-  gameHeight = 800
-  brickHeight = 40
-  paddleHeight = 10
-=======
-  brickWidth, brickHeight, speedRatio, paddleStep :: Float
-  brickWidth = 100
-  brickHeight = 40
-  speedRatio = 1.5
-  paddleStep = 5
->>>>>>> Better collision with paddle and walls limit
+-- | all the heights : brick, game, paddle
+brickHeight, gameHeight, paddleHeight :: Height
+brickHeight = 40
+gameHeight = 800
+paddleHeight = 20
 
-  -- | Radius of the ball
-  ballRadius :: Radius
-  ballRadius = 10
+-- | Radius of the ball
+ballRadius :: Radius
+ballRadius = 10
 
-  -- | All the wall positions
-  wallUpPos, wallDownPos, wallLeftPos, wallRightPos :: Position
-  wallUpPos   = (0, gameHeight / 2)      -- ^ top wall position
-  wallDownPos = (0,-(gameHeight/ 2))   -- ^ botom wall position
-  wallLeftPos = (gameWidth / 2 , 0)      -- ^ left wall position
-  wallRightPos = (-(gameWidth / 2), 0)  -- ^ right wall position
+-- | All the wall positions
+wallUpPos, wallDownPos, wallLeftPos, wallRightPos :: Position
+wallUpPos    = (0, gameHeight / 2)     -- ^ top wall position
```

Comment:
Personal taste, again: I have come to hate vertical alignments that depend on the length of, e.g., identifiers. These are a pain, as you regularly need to address the alignment when adding or renaming something.